polyfill/test/parse.mjs

```diff
@@ -0,0 +1,49 @@
+#! /usr/bin/env -S node --experimental-modules
+
+/*
+ ** Copyright (C) 2020 Igalia S.L. All rights reserved.
+ ** This code is governed by the license found in the LICENSE file.
+ */
+
+import Demitasse from '@pipobscure/demitasse';
+const { describe, it, report } = Demitasse;
+
+import Pretty from '@pipobscure/demitasse-pretty';
+const { reporter } = Pretty;
+
+import Assert from 'assert';
+const { ok: assert, strictEqual: equal, deepStrictEqual: dEqual, throws } = Assert;
+
+import * as Temporal from 'tc39-temporal';
+
+const sample = '2019-10-29T10:46:38.271986102Z+01:00';
```

Comment:
See comment above about `Z` — this isn't a valid ISO string. `Z` and the time zone are mutually exclusive.

Reply:
My bad, yeah.